index.ts

```diff
@@ -188,12 +205,29 @@ export async function getGitHubProfile() {
  */
 
 // Membrane's core structure is represented as a graph, where each Membrane program is a node in the graph.
-// This program is a node with other nodes in its subgraph, like the `ping` action.
+// This `getting-started` program is a node with other nodes in its subgraph, like the `ping` action.
 // This program also connects to other nodes in the graph, like `email`, `sms`, and `github`.
 
 // A Membrane program's graph is represented by its Schema.
-// You can view and update your program's schema in the bottom right panel of the Membrane Explorer.
+// You can view and update your program's schema in the CONFIG pannel (bottom of the Explorer sidebar).
 // When you update a program's schema in the explorer, the program's `memconfig.json` file will automatically update.
 // You should never have to manually edit `memconfig.json` nor `memconfig.lock`.
 
-// To learn more about the Membrane graph, visit: https://docs.membrane.io/concepts/the-graph/
+// 🔗 To learn more about the Membrane graph, visit: https://docs.membrane.io/concepts/the-graph/
+
+/**
+ * ========================================================================================================================
+ * 🎉 Well done! 🎉
+ *
+ * Thanks for going through our getting-started tour!
+ * We know there's a lot to wrap your head around, and we want to make learning and using Membrane _way_ simpler.
+ *
+ * On that note, we'd love your feedback. You can reach us:
+ * 1. via this form (powered by a Membrane HTTP endpoint): https://spare-346-sector-257-manner-983-bet.hook.membrane.io
+ * 2. via email: juan@membrane.io, justin@membrane.io, pete@membrane.io
```

Comment:
```suggestion
 * 2. via email: contact@membrane.io
```

Let's use the shared google group so it goes to all of us.